number-of-agent-in-queue/nodejs/index.js

```diff
@@ -0,0 +1,108 @@
+// Set Genesys cloud objects
+const platformClient = require('purecloud-platform-client-v2');
+const client = platformClient.ApiClient.instance;
+
+// Import library to use for input in NPM
+const prompt = require('prompt');
+
+// Get client credentials from environment variables
+const CLIENT_ID = process.env.GENESYS_CLIENT_ID;
+const CLIENT_SECRET = process.env.GENESYS_CLIENT_SECRET;
+
+// Instantiate API
+let routingApi = new platformClient.RoutingApi();
+
+// Declare global variables
+let queueId = "";
+let queueName = "";
+
+
+// Properties of input
+let schema = {
+  properties: {
+    queueName: {
+      message: 'Name of queue',
+      required: true
+    }
+  }
+};
+
+// Start the prompt
+function inputQueue() {
+  prompt.start();
+  prompt.get(schema, function (_err, result) {
+    queueName = result.queueName
+    listQueues(queueName)
+  });
+}
+
+// Display all the queues in the org base on the parameters
+function listQueues(queueName) {
+  let opts = {
+    'pageSize': 100, // Number | Page size
+    'pageNumber': 1, // Number | Page number
+    'sortBy': "name", // String | Sort by
+    'name': queueName // String | Name
+  };
+
+  routingApi.getRoutingQueues(opts)
+    .then((routingQueuesData) => {
+      console.log(`getRoutingQueues success! data: ${JSON.stringify(routingQueuesData, null, 2)}`);
+      checkEntities(routingQueuesData);
+    })
+    .catch((err) => {
+      console.log('There was a failure calling getRoutingQueues');
+      console.error(err);
+    });
+
+}
+
+// Check for the number of entities returned and Search for the routing id of the queue
+function checkEntities(routingQueuesData) {
+
+  return (routingQueuesData.entities).length < 1 ? console.log("Queue not found.") :
```

Comment:
While I can personally appreciate nested ternary statements, it requires some effort to read. :) Can you break this up into multiple statements? It'll be more verbose, but easier to comprehend.

Reply:
Done with the said changes.